_i18n/it/resources/moneropedia/account.md

```diff
@@ -2,37 +2,46 @@
+Il tuo account è basato su due coppie di chiavi, la coppia relativa alla "chiave di spesa" (@spend-key) e la coppia relativa alla "chiave di vista" (@view-key). La chiave di spesa è speciale in quanto viene richiesta per spendere i fondi Monero, mentre la chiave di vista ti consente di rivelare le tue @transazioni a terzi, ad esempio per scopi di controllo (audit) o contabilità. Queste chiavi, utilizzate nel tuo account, svolgono anche un ruolo molto importante nella [privacy delle transazioni](#privacy-transazioni) di Monero.
 
-The private keys for both of these must be protected by you in order to retain your account privacy. On the other hand, the public keys are obviously public (they are part of your Monero account address). For normal public/private key cryptography someone could send you a private message by encrypting it with either of your public keys, and you would then be the only one able to decrypt it with your private keys.
+Le chiavi private di spesa e di vista devono essere protette da te al fine di mantenere intatta la privacy del tuo account; le chiavi pubbliche di spesa e di vista, invece, sono ovviamente pubbliche (fanno parte dell'indirizzo del tuo account Monero). Usando la normale crittografia a chiave pubblica/privata, qualcuno potrebbe inviarti un messaggio privato cifrandolo con una delle tue chiavi pubbliche e tu saresti l'unico in grado di decifrarlo con le tue chiavi private.
 
-### Backing Up Your Account
 
-When you manage your own Monero Account with the private @spend-key, you are solely responsible for the security of your funds. Thankfully, Monero makes it very easy to backup your account. When creating a Monero account for the first time you will be given a unique @mnemonic-seed for your account that consists of 13 or 25 words in the language of your choosing. **This seed is the only thing you need to backup for your account**, and so it is imperative that it is written down and stored securely.  Never store this seed in a form or location that would allow someone else to see it!
+### Fare il Backup del tuo Account
+
+Quando gestisci il tuo account Monero con la chiave di spesa (@spend-key) privata, sei l'unico responsabile della sicurezza dei tuoi fondi. Fortunatamente, Monero rende molto semplice effettuare il backup del tuo account. Quando crei un account Monero per la prima volta ti verrà dato un [seed mnemonico](#mnemonic-seed) unico per il tuo account che consiste di 13 o 25 parole nella lingua di tua scelta. **Questo seed è l'unica cosa di cui hai bisogno per eseguire il backup del tuo account **, quindi è assolutamente necessario, una volta comunicatoti, annotarlo ed archiviarlo in modo sicuro. Non conservare mai questo seed in una forma o in un luogo tale da consentire a qualcun altro di vederlo!
```

Comment:
Remove the space between `account` and `**` or the format will break

Reply:
Fixed.